setup/boards/nrf52840_pca10059.conf

```diff
@@ -1,2 +1,2 @@
-# Using first 2 flash areas from storage slot for Settings subsystem
-CONFIG_SETTINGS_FCB_NUM_AREAS=2
+# Using all 4 flash areas from storage slot for Settings subsystem
```

Comment:
I suggest you leave **pages** or **areas** in both DTS files.

Reply:
Done

Reply:
Done